node/core/av-store/src/lib.rs

```diff
@@ -1276,9 +1287,19 @@ fn store_available_data(
 		},
 	};
 
+	let erasure_span = jaeger::Span::new(candidate_hash, "erasure-coding")
+		.with_candidate(candidate_hash)
+		.with_pov(&available_data.pov);
+
```

Comment:
Now that this is consensus-critical, let's add a comment here that this check is relied upon for candidate validity.